backend/runtests.sh

```diff
@@ -7,27 +7,27 @@ function poetry_func () {
 
 function lint() {
   echo "Lint backend code..."
-  ruff --exclude hunter $lintargs
+  ruff check --exclude hunter $lintargs .
```

Comment:
This could be a separate PR... Just because we have ai doesn't mean it's good practice.